cancel-in-progress/child_workflow.go

```diff
@@ -0,0 +1,19 @@
+package cancel_in_progress
+
+import (
+	"go.temporal.io/sdk/workflow"
+	"time"
```

Comment:
I forget whether it's `go fmt` or `goimports` but may need to run them both so that stdlib imports are separate from others